expat/configure.ac

```diff
@@ -221,6 +221,8 @@ AC_DEFINE([XML_DTD], 1,
           [Define to make parameter entity parsing functionality available.])
 AC_DEFINE([XML_DEV_URANDOM], 1,
           [Define to include code reading entropy from `/dev/urandom'.])
+AC_DEFINE([XML_ATTR_INFO], 1,
+          [Define to allow retrieving the byte offsets for attribute names and values.])
```

Comment:
This code force-enables `XML_ATTR_INFO` when compiling with GNU Autotools. The CMake version above has a switch, in contrast. It would be nice to have a switch on both sides.

Reply:
Update review comments in the change